test/ch/imedias/rsccfx/model/xml/SupporterHelperTest.java

```diff
@@ -0,0 +1,146 @@
+ */
+public class SupporterHelperTest {
+
+  public static final int DEFAULT_SUPPORTER_SIZE = 7;
+  SupporterHelper supporterHelper;
+  Rscc mockModel;
+  String supportersXml;
+  Supporter supporter1;
+  Supporter supporter2;
+  Supporter supporterLast;
+  List<Supporter> expectedSupporters;
+
+  /**
+   * Initializes test fixture before each test.
+   */
+  @Before
+  public void setUp() {
+    mockModel = mock(Rscc.class);
+    supporterHelper = new SupporterHelper(mockModel);
+
+    String pathToDefaultSupporters = Rscc.REMOVE_FILE_IN_PATH.apply(
+        getClass().getClassLoader().getResource(Rscc.DEFAULT_SUPPORTERS_FILE_NAME).getFile()
+    );
+    when(mockModel.getPathToDefaultSupporters()).thenReturn(pathToDefaultSupporters);
+
+    supportersXml = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n"
+        + "<supporters>\n"
+        + "    <supporter>\n"
+        + "        <address>agora.imedias.ch</address>\n"
+        + "        <chargeable>false</chargeable>\n"
+        + "        <description>imedias</description>\n"
+        + "        <encrypted>false</encrypted>\n"
+        + "        <port>5500</port>\n"
+        + "    </supporter>\n"
+        + "    <supporter>\n"
+        + "        <address>agora.imedias.ch</address>\n"
+        + "        <chargeable>false</chargeable>\n"
+        + "        <description>imedias (encrypted)</description>\n"
+        + "        <encrypted>true</encrypted>\n"
+        + "        <port>50000</port>\n"
+        + "    </supporter>\n"
+        + "</supporters>\n";
+
+
+    supporter1 = new Supporter("imedias", "agora.imedias.ch", "5500",
+        false, false);
+    supporter2 = new Supporter("imedias (encrypted)", "agora.imedias.ch", "50000",
+        true, false);
+    expectedSupporters = new ArrayList<>();
+    expectedSupporters.add(supporter1);
+    expectedSupporters.add(supporter2);
+    supporterLast = new Supporter();
+  }
+
+  /**
+   * Test for {@link SupporterHelper#loadSupporters()}.
+   */
+  @Test
+  public void testLoadSupporters() throws Exception {
+    // if preferences are null, test if default supporters are returned
+    supporterHelper.setSupportersInPreferences(null);
+    List<Supporter> actualSupporters = supporterHelper.loadSupporters();
+    verifyDefaultSupporters(actualSupporters);
+    actualSupporters = null;
+
+    // if preferences are not null
+    supporterHelper.setSupportersInPreferences(supportersXml);
+    actualSupporters = supporterHelper.loadSupporters();
+    assertEquals(2, actualSupporters.size());
+    assertEquals(supporter1, actualSupporters.get(0));
+    assertEquals(supporter2, actualSupporters.get(1));
+    actualSupporters = null;
+
+    // if preferences contain invalid xml
+    String invalidXml = "<test>this is not a correctly formatted xml</not test>";
+    supporterHelper.setSupportersInPreferences(invalidXml);
+    actualSupporters = supporterHelper.loadSupporters();
+    verifyDefaultSupporters(actualSupporters);
```

Comment:
Since you're loading a wrong XML file... shouldn't this test then fail instead of pass?